bin/configurer/users.json

```diff
@@ -0,0 +1,16 @@
+[
+  {"email": "damian@swansea.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "kurt@swansea.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "james@swansea.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "sam@hillingdon.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "siva@hillingdon.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "raghu@swindon.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "sam@swindon.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "raghu@wiltshire.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "sam@wiltshire.gov.uk", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-solicitor", "lastName": "(local-authority)"},
+  {"email": "hmcts-admin@example.com", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-courtadmin", "lastName": "(hmcts-admin)"},
+  {"email": "cafcass@example.com", "roles": "caseworker,caseworker-publiclaw,caseworker-publiclaw-cafcass", "lastName": "(cafcass)"},
+  {"email": "judiciary@mailnesia.com", "roles": "caseworker-publiclaw-courtadmin,caseworker,caseworker-publiclaw,caseworker-publiclaw-judiciary,caseworker-publiclaw-gatekeeper", "lastName": "(judiciary)"},
```

Comment:
List of roles is bit messy here, it starts with more concrete role, then there are more generic roles and then more specific roles again. Every time I need to look into that it gives me headache. Might be an opportunity to clean that up.

Reply:
done